src/main/java/training/bookish/models/Book.java

```diff
@@ -0,0 +1,87 @@
+package training.bookish.models;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    private Integer id;
+    private String isbn;
+    private String title;
+    private Integer numberOfCopies;
+    private String author;
+    private Set<Loan> loans;
+
+    public Book() {}
+
+    public Book(int id, String isbn, String title, String author, int numberOfCopies) {

```

Comment:
When writing models that need getters and setters (e.g. because the ORM requires it), it's common to not add a constructor with parameters. Just use the setters when you to create a new instance.

I don't think this version is _wrong_ and I don't think it's worth changing, but it's worth being aware that it's not very idiomatic for Java data models.